qemu/tests/cfg/netperf_stress_test.cfg

```diff
@@ -16,8 +16,10 @@
     disable_firewall = "service iptables stop; systemctl stop firewalld.service"
     deviation_time = 20
     Windows:
-        netperf_server_link_win = "netserver-2.6.0.exe"
-        netperf_client_link_win = "netperf.exe"
+        netperf_server_link_win_64bit = "netperf-win\netserver.exe"
+        netperf_client_link_win_64bit = "netperf-win\netperf.exe"
+        netperf_server_link_win_32bit = "WIN_UTILS:\netperf-win\netserver.exe"
+        netperf_client_link_win_32bit = "WIN_UTILS:\netperf-win\netperf.exe"
```

Comment:
```suggestion
        Windows..32:
            netperf_server_link_win = "WIN_UTILS:\netperf-win\netserver.exe"
            netperf_client_link_win = "WIN_UTILS:\netperf-win\netperf.exe"
        Windows..64:
            netperf_server_link_win = "netperf-win\netserver.exe"
            netperf_client_link_win = "netperf-win\netperf.exe"
```

Reply:
Cool!!! Best example for me. Thanks Yihuang.